packages/nx-monorepo/src/nx-monorepo.ts

```diff
@@ -337,6 +347,14 @@ export class NxMonorepoProject extends TypeScriptProject {
     }
   }
 
+  /**
+   * Add an additional package glob to the workspace.
+   * @param packageGlob path to the package to include in the workspace (for example packages/my-package)
+   */
+  public addWorkspacePackage(packageGlob: string) {
```

Comment:
Can we support vararg so multiple can be added in one call?

Reply:
Yeah good idea!